classifier.py

```diff
@@ -1685,8 +1693,15 @@ class KeywordBasedClassifier(AgeOrGradeClassifier):
     JUVENILE_INDICATORS = match_kw(
         "for children", "children's", "juvenile",
         "nursery rhymes", "9-12")
-    YOUNG_ADULT_INDICATORS = match_kw("young adult", "ya", "12-Up", 
-                                      "teenage fiction")
+    YOUNG_ADULT_INDICATORS = match_kw(
```

Comment:
I noticed that keywords based on 'teen' language were not being treated as indicating Young Adult even when that was clearly the case.